designer/client/conditions/inline-conditions-absolute-dates.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react'
+import { Value } from './inline-condition-values'
```

Comment:
I specifically wanted the Value class rather than the ConditionValue abstract type here, Perhaps I need to rename Value to SimpleConditionValue? Or rename ConditionValue to AbstractConditionValue and this to ConditionValue?

Reply:
whichever you think is best. you can keep the class name as Value, but just change the reference import 'as whateverValue' for clarity

Reply:
done